src/ifElse.test.js

```diff
@@ -1,11 +1,88 @@
 'use strict';
 
-describe('ifElse', () => {
-  // const { ifElse } = require('./ifElse');
+const { ifElse } = require('./ifElse');
 
-  it('should ', () => {
+describe(`'ifElse' function:`, () => {
+  it(`should call a 'first' callback `
+      + `if 'condition' returns boolean 'true'`, () => {
+    const condition = jest.fn(() => true);
+    const firstCb = jest.fn(() => {});
+    const secondCb = jest.fn(() => {});
 
+    ifElse(condition, firstCb, secondCb);
+
+    expect(firstCb).toHaveBeenCalled();
+  });
+
+  it(`should return nothing`, () => {
+    const condition = jest.fn(() => true);
+    const firstCb = jest.fn(() => {});
+    const secondCb = jest.fn(() => {});
+
+    const isElse = ifElse(condition, firstCb, secondCb);
+
+    expect(isElse).toBeUndefined();
   });
 
```

Comment:
I think, it would be nice to add one more test to check that the first callback wasn't called if the condition callback returns 'false', for instance:

const condition = jest.fn(() => false);
const firstCb = jest.fn(() => {});
const secondCb = jest.fn(() => {});

ifElse(condition, firstCb, secondCb);

expect(firstCb).not.toHaveBeenCalled();